components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,185 @@
+                height={28}
+                src="/static/images/updates-and-news-modal-icon.svg"
+                alt="Updates and News Icon"
+              />
+            </Span>
+            <Text fontSize={['25px', '28px']}>News and Updates</Text>
+          </Flex>
+        </Flex>
+      </ModalHeader>
+      <hr />
+      <ModalBody>
+        <Query
+          query={newsAndUpdatesQuery}
+          variables={{ collectiveSlug: 'opencollective', onlyChangelogUpdates: true }}
+          context={API_V2_CONTEXT}
+        >
+          {({ data, loading }) =>
+            loading === false && data ? (
+              <StyledCarousel contentPosition="left">
+                {data.account.updates.nodes.map(update => (
+                  <Container key={update.id}>
+                    <PublishedDate>
+                      <FormattedDate value={update.publishedAt} day="numeric" month="long" year="numeric" />
+                    </PublishedDate>
+                    <Flex>
+                      <Span paddingTop="10px">
+                        <NextIllustration
+                          width={12}
+                          height={12}
+                          src="/static/images/news-and-updates-ellipse.svg"
+                          alt="News and Updates Ellipse"
+                        />
+                      </Span>
+                      <Text fontSize="20px" margin="0px 12px">
+                        {update.title}
+                      </Text>
+                    </Flex>
+                    <Flex pt={2} pb={3}>
+                      <StyledLink
+                        onClick={event => goToUpdatePage(router, `/opencollective/updates/${update.slug}`, event)}
+                        color="#1153D6"
+                        fontSize="14px"
+                        display="flex"
+                      >
+                        <FormattedMessage
+                          id="NewsAndUpdates.link.giveFeedback"
+                          defaultMessage="Read more & give Feedback"
+                        />
+                      </StyledLink>
+                    </Flex>
+                    <Flex pb={1}>
+                      <HTMLContent color="#313233" mt={1} fontSize="16px" content={update.summary} />
+                    </Flex>
+                    <Flex pb={3}>
+                      {update.summary.slice(update.summary.length - 3) === '...' && (
+                        <StyledLink
+                          onClick={event => goToUpdatePage(router, `/opencollective/updates/${update.slug}`, event)}
+                          fontSize="14px"
+                          display="flex"
+                        >
+                          <FormattedMessage id="NewsAndUpdates.link.readMore" defaultMessage="Read more" />
+                        </StyledLink>
+                      )}
+                    </Flex>
+                  </Container>
+                ))}
+              </StyledCarousel>
+            ) : null
```

Comment:
Would be great to show a loading spinner (`Loading`) or a `LoadingPlaceholder` rather than a blank space